runners/flink/src/test/java/org/apache/beam/runners/flink/FlinkExecutionEnvironmentsTest.java

```diff
@@ -279,33 +279,45 @@ public void shouldDetectMalformedPortStreaming() {
     options.setFlinkMaster("host:p0rt");
 
     expectedException.expect(IllegalArgumentException.class);
-    expectedException.expectMessage("Provided port is malformed");
+    expectedException.expectMessage("Unparseable port number");
 
     FlinkExecutionEnvironments.createStreamExecutionEnvironment(options, Collections.emptyList());
   }
 
   @Test
-  public void shouldFailOnEmptyPortBatch() {
+  public void shouldSupportIPv6Batch() {
     FlinkPipelineOptions options = PipelineOptionsFactory.as(FlinkPipelineOptions.class);
     options.setRunner(FlinkRunner.class);
-    options.setFlinkMaster("host:");
```

Comment:
I removed this test which I had just merged with #7187, because `HostAndPort` permits a host with empty port. Thinking about it, I don't see why it should be a problem.